src/components/AddCourseButton.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div
+    class="semester-courseWrapper semester-addWrapper"
+    :class="{ 'semester-addWrapper--compact': compact, 'my-2 mx-0': shouldClearPadding }"
```

Comment:
where did the classes `my-2` and `mx-0` come from?

Reply:
Bootstrap https://getbootstrap.com/docs/4.0/utilities/spacing/